CHANGELOG.md

```diff
@@ -4,7 +4,7 @@ See [RELEASE](./RELEASE.md) for workflow instructions.
 
 ## Unreleased
 
-### Improvements
+### Release v1.5.2
```

Comment:
I think that's the wrong header to change for the release version - it should be the `## Unreleased`